### ISSUE TITLE
Sleepandroid 2.8 incompatible with Kodi 19

### ISSUE THREAD

**Issue by tt4079** [elmerohueso/xbmc.repo.elmerohueso#6]
I have been using this kodi addon and it works brilliantly. That is, until Kodi v19 which gives an error "Incompatible add-ons. The following add-ons are incompatible with this version of Kodi and have been automatically disabled: Android Power Options."

Will the addon be updated to work with the latest version of Kodi?

Many thanks

Wayne

**Comment by elmerohueso** [comment]
Sorry, I've moved to Roku with Plex, so this will not be updated by me.  But I welcome anyone who wants to fork it and updated it.

**Comment by tt4079** [comment]
Fair enough. I will stick with Kodi 18. I cannot do without your addon :-)

Cheers
